docusaurus/docs/guides/deployments/10-linux/20-router/10-deploy.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Router Deployment
+---
+
+import CliLogin from '/docs/_cli-login.md'
+import LinuxPackageRepo from '/docs/_linux-package-repo.mdx';
+import CliLogFormats from '/docs/_cli-log-levels-and-formats.md'
+
+This article is about deploying a Ziti router as a Linux system service. [This page](/learn/introduction/03-components.md#openziti-router) provides an overview of the architecture.
```

Comment:
i didn't catch this before, but I'd prefer to see us use the "informal" terminology in most every case. Instead of `Ziti router` or `Ziti controller` they should be deformalized to just "router" and just "controller". I scanned through and there's a lot of places where the more formal wording is used. I'd like to turn them into the less formal tone. It struck me today when I was looking at the "Router Deployment" doc and "Ziti router" was front and center. Also, when using the formal name, I think it's best to treat it like a product name and thus it's a proper noun so both Z and R get capitalized.

Reply:
https://github.com/openziti/ziti-doc/pull/854